test/config_test.cpp

```diff
@@ -391,6 +391,10 @@ void ConfigTest::testSchemaDirectory()
 
 void ConfigTest::testLogFileRollover()
 {
+	// This test/rollover is fragile on Windows due to file caching.
```

Comment:
I really tried to fix the behaviour on Windows (calling flush etc...) but I could not get Windows to not use write caching when using the C-Runtime methods to open the file. This could be done by rewriting the RollingFileLogger to use specific Windows API when opening the file but this seemed too much work. Eventually Windows does flush contents to disk and the log file rolls over, but not at a determinate file-size (asserted by this test).